mappings/net/minecraft/tag/TagGroup.mapping

```diff
@@ -8,13 +8,41 @@ CLASS net/minecraft/class_5414 net/minecraft/tag/TagGroup
 		ARG 1 object
 	METHOD method_30207 create (Ljava/util/Map;)Lnet/minecraft/class_5414;
 		ARG 0 tags
-	METHOD method_30208 toPacket (Lnet/minecraft/class_2378;)Lnet/minecraft/class_5414$class_5748;
+	METHOD method_30208 pack (Lnet/minecraft/class_2378;)Lnet/minecraft/class_5414$class_5748;
+		COMMENT Packs this tag group for serialization.
+		ARG 1 registry
 	METHOD method_30210 getTag (Lnet/minecraft/class_2960;)Lnet/minecraft/class_3494;
 		ARG 1 id
 	METHOD method_30211 getTagIds ()Ljava/util/Collection;
 	METHOD method_30213 getTagOrEmpty (Lnet/minecraft/class_2960;)Lnet/minecraft/class_3494;
 		ARG 1 id
 	METHOD method_30214 createEmpty ()Lnet/minecraft/class_5414;
+	METHOD method_33155 unpack (Lnet/minecraft/class_5414$class_5748;Lnet/minecraft/class_2378;)Lnet/minecraft/class_5414;
+		COMMENT Unpacks a packed tag group, usually obtained from serialization.
+		ARG 0 packed
+		ARG 1 registry
+	METHOD method_33156 (Lnet/minecraft/class_2378;Ljava/util/Map;Lnet/minecraft/class_2960;Lnet/minecraft/class_3494;)V
+		ARG 2 id
+		ARG 3 tag
+	METHOD method_33157 (Lnet/minecraft/class_2378;Ljava/util/Map;Lnet/minecraft/class_2960;Lit/unimi/dsi/fastutil/ints/IntList;)V
+		ARG 2 id
+		ARG 3 entries
 	CLASS 1
 		FIELD field_25742 tags Lcom/google/common/collect/BiMap;
 		FIELD field_25743 emptyTag Lnet/minecraft/class_3494;
+	CLASS class_5748 Packed
```

Comment:
I think "packed" is a bad name for this, it doesn't reflect the purpose of the class. The reason this class exists is to contain the block/item/fluid/entity type/game event ids rather than the instances of those classes, because packets are deserailized on the network thread rather than the game thread, and accessing the dynamic registry manager from the network thread is unsafe (indeed, the join game packet where the dynamic registry manager is created may not have even been processed yet when the synchronize tags packet is deserialized).

Reply:
It indeed is a serialized form. I thought `Packed` can mean `Serialized`. Should I name this `Serialized`? I feel like there can be more concise name. What's your suggestion?

Reply:
In multiconnect I have anticipated that this class will be named something along the lines of `NetworkData` (as in, I name variables in anticipation of the class being mapped before it's actually mapped). `Packed` doesn't mean `Serialized` to me. Serialization is converting program data to a standardized format that can be read and understood by other programs or other instances of the same program. Packed to me generally means reducing the amount of data something takes up, usually in the form of using the same integer/long to store multiple values.

Reply:
Then I will just use `Serialized`.